### PR TITLE
Fix for removing bad stereo indications from Atropisomer parsing

### PR DESCRIPTION
**Pull request by tadhurst-cdd**
#### Reference Issue
When parsing some atropisomers with atomatic rings, a wedge bond can produce more than one potential atropisomer bond.  One is not valid, and should be removed.  The code that should remove this was throwing an error because the bond type (aromatic) cannot be an atropisomer.  Consider the following structure:
.../Code/GraphMol/FileParsers/test_data/atropisomers/BMS-986142_atrop1.sdf

BMS-986142_atrop1

The bond in the ring that is shown as single is also perceived as a potential  atropisomer when the new stereo chemistry is used.  Later, this ring is aromatized, and this causes the error thrown in the uncorrected code.

### CODE REVIEW

**Comment by greglandrum** [comment]
Hi @tadhurst-cdd please provide a short reproducible (in the PR description would be fine) that triggers the problem.
I am probably misunderstanding your description of the problem, but working from master, everything seems ok:
```
In [12]: Chem.SetUseLegacyStereoPerception(False)

In [13]: m = Chem.MolFromMolFile('../Code/GraphMol/FileParsers/test_data/atropisomers/BMS-986142_atrop1.sdf')

In [14]: m.GetNumAtoms()
Out[14]: 42

In [15]: m = Chem.MolFromMolFile('../Code/GraphMol/FileParsers/test_data/atropisomers/BMS-986142_3d.sdf')
[09:30:58] Warning: molecule is tagged as 2D, but at least one Z coordinate is not zero. Marking the mol as 3D.

In [16]: m.GetNumAtoms()
Out[16]: 42

In [17]: m = Chem.MolFromMolFile('../Code/GraphMol/FileParsers/test_data/atropisomers/BMS-986142.sdf')

In [18]: m.GetNumAtoms()
Out[18]: 42
```

**Comment by tadhurst-cdd** [comment]
The changes made in this PR are already in master from a recent change.  I am closing this PR

**Comment by tadhurst-cdd** [comment]
PR is obsolete - Production has already been fixed

File: Code/GraphMol/FindStereo.cpp [hunk 1]
```diff
@@ -225,7 +225,7 @@ StereoInfo getStereoInfo(const Bond *bond) {
           UNDER_CONSTRUCTION("unrecognized bond stereo type");
       }
     } else {
-      sinfo.specified = Chirality::StereoSpecified::Unspecified;
+      UNDER_CONSTRUCTION("unrecognized bond stereo type");
```
Comment (greglandrum) [inline Code/GraphMol/FindStereo.cpp:228]: This change seems wrong; it leads to:
```
In [8]: m = Chem.MolFromSmiles('CC=CC')

In [9]: Chem.FindPotentialStereo(m,cleanIt=True,flagPossible=True)
[06:40:00] 

****
Incomplete Code
This routine is still under development
Violation occurred on line 228 in file /scratch/RDKit_git/Code/GraphMol/FindStereo.cpp
Failed Expression: unrecognized bond stereo type
----------
```
and is responsible for the CI build failures on this PR.
If you revert this change I think the CI tests will all pass (at least it worked on my linux box)

**Review by greglandrum — CHANGES_REQUESTED** [review]
one suggested change to fix a mistake

File: Code/GraphMol/FindStereo.cpp (hunk 1 above)
Comment (tadhurst-cdd) [reply Code/GraphMol/FindStereo.cpp]: Yes this was a mistake.  It has been restored as suggested.

**Review by greglandrum — APPROVED** [review]
LGTM

**Pull request merged**